.gitignore

```diff
@@ -1,17 +1,13 @@
 # IDE
-.vscode
+.vs
```

Comment:
This should have been added in addition to everything that was already here.

A lot of important things have been removed from this file and should be put back.

Reply:
added the others back.